### PR TITLE
Support for deploying multiple static sites/apps in one uWSGI process

### PR DESCRIPTION
**Pull request by erickpeirson**
This adds support for deploying static sites/apps under a single uWSGI process. This works by:

- Leveraging Flask's "instance directory" mechanism for app-specific config
- Adds a Helm chart that supports multiple ingress rules for the same service

Incidentally, it adds click bling to the build script.

### CODE REVIEW

File: arxiv/marxdown/factory.py [hunk 1]
```diff
@@ -59,14 +69,12 @@ def create_web_app(build_path: Optional[str] = None,
...
-                with_search=with_search
-            )
+            routes.get_blueprint(build_path, with_search=with_search)
         )
 
-    app.jinja_env.filters['format_datetime'] = format_datetime
-    app.jinja_env.filters['simepledate'] = simepledate
-    app.jinja_env.filters['pretty_path'] = pretty_path
+    app.jinja_env.filters['format_datetime'] = format_datetime  # pylint: disable=no-member
+    app.jinja_env.filters['simepledate'] = simepledate  # pylint: disable=no-member
```
Comment (mhl10) [inline arxiv/marxdown/factory.py]: Did you mean:
```suggestion
    app.jinja_env.filters['simpledate'] = simpledate  # pylint: disable=no-member
```

Comment (erickpeirson) [reply arxiv/marxdown/factory.py]: Aha! Looks like the typo was already there (I just added the pylint flag)... looks like the typo is everywhere, so I'll fix globally

Comment (erickpeirson) [reply arxiv/marxdown/factory.py]: Fixed in 04209d6

**Comment by erickpeirson** [comment]
> Looks great. With these changes, I should probably look at cleaning up the deployment for apache/mod_wsgi.

Actually, I don't think you'll need to change anything. This really just impacts deployment with uWSGI/Docker -- should be fully backward compatible, too

**Pull request merged**